.github/julia/runtests_uno_filtersqp.jl

```diff
@@ -0,0 +1,120 @@
+# Copyright (c) 2025: Charlie Vanaret and contributors
+#
+# Use of this source code is governed by an MIT-style license that can be found
+# in the LICENSE.md file or at https://opensource.org/licenses/MIT.
+
+# For help with this file, please contact `@odow` on GitHub.
+
+using Test
+
+import AmplNLWriter
+import MathOptInterface as MOI
+import MINLPTests
+import Uno_jll
+
+"""
+    Optimizer()
+
+Create a new `AmplNLWriter.Optimizer` object that uses Uno as the backing
+solver.
+"""
+function Optimizer(options)
+    return AmplNLWriter.Optimizer(Uno_jll.amplexe, options)
+end
+
+Optimizer_Uno_filtersqp() = Optimizer(["logger=SILENT", "preset=filtersqp", "QP_solver=BQPD", "max_iterations=10000", "unbounded_objective_threshold=-1e15"])
+
+# This testset runs https://github.com/jump-dev/MINLPTests.jl
+@testset "MINLPTests" begin
+    primal_target = Dict(
+        MINLPTests.FEASIBLE_PROBLEM => MOI.FEASIBLE_POINT,
+        MINLPTests.INFEASIBLE_PROBLEM => MOI.NO_SOLUTION,
```

Comment:
@cvanaret the issue is here. JuMP expects that if Uno has found an infeasible problem, it will not return a primal solution.

(See the comment in your filterslp above)

Reply:
Got it. I saw the comment and tried both `MOI.NO_SOLUTION` and `MOI.INFEASIBLE_POINT` but it made no difference. I'll just skip the test. Thanks!

Reply:
It didn't make a difference because the value was UNKNOWN_RESULT_STATUS